integrated_channels/canvas/migrations/0038_remove_canvasenterprisecustomerconfiguration_client_id_and_more.py

```diff
@@ -0,0 +1,21 @@
+# Generated by Django 4.2.15 on 2024-08-30 09:58
```

Comment:
shouldn't this migration be included in the next release?

Reply:
otherwise we'll end up with a deployment/migration conflict.

Reply:
@sameeramin ,,  @MueezKhan246 can share his experience. He has done similar work recently.

Reply:
@hamzawaleed01 we will be making separate releases for removing columns from model and then for adding migration for that removal. reference issue can get raised otherwise.